beast/tools/symlog.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+__all__ = ["symlog", "inverse_symlog"]
+
+
+symlog_linthreshold = 1e-20
+
+
+def symlog(y, C=symlog_linthreshold):
+    """
+    Compute the symlog value that transitions between linear and log scales
+    based on the theshold value C.  Symlog works for both positive and
+    negative input values.
+
+    Parameters
+    ----------
+    y : float
+        intput value
+
+    C : float
+        value of the threshold where the symlog changes from linear to log
+
+    Returns
+    -------
+    y_symlog : float
+        symlog of y
+    """
+    return np.sign(y) * np.log10(1.0 + np.abs(y / C))
```

Comment:
I'm not sure this is an issue anymore given that C is 1e-20.  We shouldn't see anything really small anymore.  This should be a separate issue so that the math can be checked.

Reply:
Fair point.  I just did a quick test with y=1e-30, and the difference in the two methods is 0.0000008%, so I agree that it's fine!